src/php/Config/PhelBuildConfig.php

```diff
@@ -84,7 +86,12 @@ public function setMainPhpPath(string $path): self
     public function getMainPhpPath(): string
     {
         if ($this->mainPhpPath !== '') {
-            return $this->mainPhpPath;
+            $explode = explode('/', $this->mainPhpPath);
+            if (count($explode) === 2) {
+                return $this->mainPhpPath;
+            }
+
+            return sprintf('out/%s', $this->getPhpFilename());
```

Comment:
`out` is specified as a literal. Isn't it `$this->getDestDir()`?
If that were the case, I think line 94 would be unnecessary since it would be the same as line 97.

Reply:
you are right, good catch! This line is not needed